sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -276,3 +276,16 @@ select luhn_check(6011111111111117);
 select luhn_check(6011111111111118);
 select luhn_check(123.456);
 
+--utf8 string validation
+select is_valid_utf8('');
+select is_valid_utf8('abc');
+select is_valid_utf8(x'80');
+select make_valid_utf8('');
+select make_valid_utf8('abc');
+select make_valid_utf8(x'80');
+select validate_utf8('');
+select validate_utf8('abc');
+select validate_utf8(x'80');
+select try_validate_utf8('');
+select try_validate_utf8('abc');
+select try_validate_utf8(x'80');
```

Comment:
I don't this is testing the codegen part of the expression.
Can we have some test that codegen works as expected?

Reply:
we have unit tests that cover both interepreted and codegen paths in StringExpressionsSuite

these are just e2e sql tests